_posts/2018-03-25-this-week-in-rustfest-3-train-strike-cfp-keynote-speakers.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "These weeks in RustFest 3: Train Strike, CfP Progress & Keynote Speakers"
+authors:
+  - dns2utf8
+---
+
+# Train Strike
+
+- Normal days
+- What happens on strike days?
+    - Will prices rise?
+    - What methods of transport are affected/available?
+
+# CfP Progress
+
+Thank you all for your 74 talks and 8 workshop submissions.
+If you remember [our post from beginning of march](/this-week-in-rustfest-1-anonymouse-abstract-first-batch-locals) we are currently in the stage "The selection".
+
+This means we have a preliminary schedule featuring:
+
+| Kind of talk  | amount planed |
+| ------------- | -------------:|
+| Keynote       | 2             |
+| 15min format  | 2             |
+| 30min format  | 9             |
+
+Over the next days we will confirm the speakers and finalize the schedule.
+
+## Workshops
+
+Currently we are gathering more space for the workshops.
+We will inform you as quickly as possible when we know who we can host.
+
+# Keynote Speakers
+
+With the `epoch = "2018"` comming up the [rust core](https://blog.rust-lang.org/2018/03/12/roadmap.html) focuses on productivity.
+That is why we are very excited to announce Matt talking about the community in Nairobi, Kenya and Vaidehi talking about the basics of computer science in a beginner friendly way.
+
+## Matt Gattou
+
+Matt is a software developer at [JUMO.world](https://jumo.world/) in Nairobi. He has a passion in building tech for social impact.
+He is also the co-organiser of [Rust Nairobi](https://www.meetup.com/de-DE/Rust-Nairobi/), and hopes to make the tech industry a better place by encouraging learning and mentorship in his community.
```

Comment:
There should not be a language indicator in the meetup URL (or it could be English).

Reply:
You are right, thanks!